### DISCUSSION TITLE
Clarify migrate:rollback command description

### DISCUSSION THREAD

**Discussion by D1360-64RC14** [laravel/framework#53844] [Ideas]
Just +1 developer that felt into that *"Rollback the last database migration"* that, actually, rolls back the previous batch; not the migration.

Reading the command description it's easy to assume it will do a `migrate:rollback --step=1`.

Maybe also add a confirmation for these destructive operations, like showing the amount of migrations or which of them will be rolled back.

I know there's a `migrate:rollback --pretend`, but no one will think of it in a local development environment.

**Comment by shaedrich** [comment]
Sounds reasonable 👍🏻  Feel free to create a PR/a couple PRs for that 😉
